webapp/sdf/.storybook/main.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  "stories": [
+    "../src/**/*.stories.mdx",
+    "../src/**/*.stories.@(js|jsx|ts|tsx)",
+    "../src/components/**/*.stories.js"
+  ],
+  "addons": [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    '@storybook/addon-a11y',
+    '@storybook/addon-controls'
+  ]
+}
```

Comment:
Just clean code improvements.
```suggestion
}